front/src/applications/operationalStudies/hooks/useSetupItineraryForTrainUpdate.ts

```diff
@@ -24,6 +24,8 @@ import { ISO8601Duration2sec } from 'utils/timeManipulation';
 
 import type { ManageTrainSchedulePathProperties } from '../types';
 
+const checkMarginValue = (margin: string) => (margin !== 'none' ? margin : '0%');
```

Comment:
Suggestion: maybe `normalizeMarginValue` would be a better name?

Reply:
Done